core/src/main/scala/org/bykn/bosatsu/pattern/SeqPart.scala

```diff
@@ -76,7 +76,7 @@ object SeqPart {
           case AnyElem => true
           case Lit(c2) =>
             p1 match {
-              case Lit(c1) => setOpsA.subset(c2, c1)
+              case Lit(c1) => setOpsA.subset(c1, c2)
```

Comment:
this was a bug. I'm surprised we never hit an issue with it before. The `SeqPattern[Set[Boolean]]` found the bug.